terraform/workspace_variables/dev.tfvars.json

```diff
@@ -23,8 +23,12 @@
     }
   },
   "alertable_apps": {
-    "find-a-lost-trn-dev": {},
-    "qualified-teachers-api-dev": {}
+    "tra-dev/find-a-lost-trn-dev": {
```

Comment:
Why do we need to increase threshold for find?

Reply:
Not needed. i thought it's same for both find and qualified teachers api?  Is it only for qualified teachers api ?

Reply:
removed it from find.